ex2/FunSetSuite.scala

```diff
@@ -0,0 +1,184 @@
+package funsets
+
+import org.scalatest.FunSuite
+
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+/**
+ * This class is a test suite for the methods in object FunSets. To run
+ * the test suite, you can either:
+ *  - run the "test" command in the SBT console
+ *  - right-click the file in eclipse and chose "Run As" - "JUnit Test"
+ */
+@RunWith(classOf[JUnitRunner])
+class FunSetSuite extends FunSuite {
```

Comment:
usually, GitHub shows the test after the "real" code, I wonder how it does it...
maybe you can you try to rename it to something like FunSetTest? or have it under test package? and then see if the tests come last...

Reply:
Ok, so I didn't want to use the course actual files and folder structure, due to this code not being mine, so I just pushed the code I wrote.

All tests pass :)

Reply:
can you try to rename to `FunSetTest` to see if github put it second to the "real" code?